pom.xml

```diff
@@ -109,6 +109,7 @@
 
     <!-- test library versions -->
     <junit.version>4.12</junit.version>
+    <hamcrest-all.version>1.3</hamcrest-all.version>
```

Comment:
if this is testing only, should it go to submodule pom file?
also we need to add this to LICENSE